control/isis.yaml

```diff
@@ -89,4 +95,43 @@ components:
           maximum: 4096
           default: 10
           minimum: 0
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
+          
+    Action.Protocol.Isis.PlannedRestart:
+      description: >-
+        Initiates IS-IS Planned Graceful Restart process for the selected IS-IS routers.
+        If no name is specified then Graceful Restart will be sent to all configured IS-IS routers.
+        When an emulated IS-IS router is in the planned "Restarting" mode,
+        it sends an IIH PDU containing a Restart TLV with the PR (Planned Restart Request) bit set and
+        set the Remaining Time with the restart_time greater than the expected control-plane restart time 
+        that is the maximum time within which this router or routers will complete the graceful restart.
+        It waits for PA (Planned Restart Acknowledge) in an IIH PDU from Neigbhor(s). 
+        The use of the PR bit provides a means to safely support restart periods that are significantly longer than standard Holding Times.
+        The PR bit SHOULD remain set in IIHs until the restart is initiated.
+        Refrence: https://datatracker.ietf.org/doc/html/rfc8706#section-3.2.3.
+        Once the Restaring Router receives the Restart Tlv with PA bit is set, it intiates the Restart Request with the RR bit is set. 
+        The holding_time is set as the Remaining Time as received in Restart Tlv or by remaining time of restart_time that was sent 
+        in the Planned Restart Request Tlv. This is left by the implementation.
```

Comment:
This is left to the choice of the implementation.

Reply:
done